scripts/init.js

```diff
@@ -23,9 +23,14 @@ module.exports = function(appPath, appName, verbose, originalDirectory) {
     appPackage.dependencies[key] = ownPackage.devDependencies[key];
   });
 
+  // Test config
+  appPackage.jest = {
+    preset: '<rootDir>/node_modules/react-scripts/config/jest',
```

Comment:
*However* I would of course prefer there be a way to pass config as a command line option or Node API argument. @cpojer can we make it so?